.github/workflows/coverity_scan.yml

```diff
@@ -0,0 +1,60 @@
+name: coverity-scan
+on:
+  workflow_dispatch: # because sometimes you just want to force a branch to have tests run
+  push:
+    branches:
+      - "**"
+    paths:
+      - "backend/**"
+      - .github/workflows/coverity_scan.yml
+  schedule:
+    - cron: '0 18 * * *' # Daily at 18:00 UTC
+
+defaults:
+  run:
+    working-directory: ./backend
+
+jobs:
+  latest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+      - name: Install dependencies
```

Comment:
Just supply the correct working-directory attribute here

Reply:
Does that work for steps? Google seems to indicate it’s only at the job level.

Reply:
It does—see https://github.com/CDCgov/prime-data-input-client/blob/15139bacd09a523e4ddf50002d30a036c49a903f/.github/workflows/smoketestBackend.yml#L116